### DISCUSSION TITLE
watch 能否实现大陆地区开通呼吸暂停和房颤历史的手段？

### DISCUSSION THREAD

**Discussion by hl312791880** [NSRingo/.github#19] [Ideas 想法]
watch 能否实现大陆地区开通呼吸暂停和房颤历史的手段？

**Comment by huyg0415** [comment]
It's on Taobao, which is a few dozen yuan. It is best to apply for a new AppleID, which is safe and reliable. And in this way, you can use the newly applied AppleID to open these functions for other watches.

**Comment by cceniam** [comment]
有，使用外置gps定位，keywords: lbs
